dotCMS/src/main/java/com/dotcms/http/CircuitBreakerUrl.java

```diff
@@ -83,18 +84,16 @@ public class CircuitBreakerUrl {
     private int response = -1;
     private Header[] responseHeaders;
     private final boolean allowRedirects;
-    private final boolean throwWhenNot2xx;
+    private final boolean throwWhenError;
```

Comment:
Renaming to include a broader concept not just for the http status code to be between 200 and 299.